homeassistant/components/scrape/config_flow.py

```diff
@@ -9,8 +9,13 @@
 import voluptuous as vol
 
 from homeassistant.components.rest import create_rest_data_from_config
-from homeassistant.components.rest.data import DEFAULT_TIMEOUT
-from homeassistant.components.rest.schema import DEFAULT_METHOD, METHODS
+from homeassistant.components.rest.data import (  # pylint: disable=hass-component-root-import
+    DEFAULT_TIMEOUT,
+)
+from homeassistant.components.rest.schema import (  # pylint: disable=hass-component-root-import
+    DEFAULT_METHOD,
+    METHODS,
+)
```

Comment:
Since rest is only used by scrape, I think it's better to use `pylint: disable` rather than expose more rest functionnality.